mod/gc_group_layout/actions/groups/retrieve_member_list.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+
+if (!elgg_is_xhr()) {
+	register_error('Sorry, Ajax only!');
+	forward();
+}
+
+$option = get_input('option');
+
+error_log('option ' . $option);
```

Comment:
Do you need the debugging output on prod?

Reply:
error logging / console logging are now removed